protocols/gossipsub/src/behaviour/tests.rs

```diff
@@ -2351,9 +2353,10 @@ fn test_add_outbound_peers_if_min_is_not_satisfied() {
 
 //TODO add a test that ensures that new outbound connections are recognized as such.
 // This is at the moment done in behaviour with relying on the fact that the call to
-// `inject_connection_established` for the first connection is done before `inject_connected`
-// gets called. For all further connections `inject_connection_established` should get called
-// after `inject_connected`.
+// `NetworkBehaviour::on_swarm_event` with `FromSwarm::ConnectionEstablished`
+// for the first connection is done before `inject_connected`
```

Comment:
The mention of `inject_connected` is outdated here, right?

Reply:
Yes, `TODO` is outdated since https://github.com/libp2p/rust-libp2p/pull/2445. Good catch @jxs. Please remove.